hyperspy/axes.py

```diff
@@ -1343,24 +1743,33 @@ def _get_dimension_str(self):
     def __repr__(self):
         text = ('<Axes manager, axes: %s>\n' %
                 self._get_dimension_str())
-        ax_signature = "% 16s | %6g | %6s | %7.2g | %7.2g | %6s "
+        ax_signature_uniform = "% 16s | %6g | %6s | %7.2g | %7.2g | %6s "
+        ax_signature_non_uniform = "% 16s | %6g | %6s |  non-uniform axis  | %6s "
```

Comment:
There is one space too much with `non-uniform axis`: the columns are not aligned between uniform and non-uniform axis.

Reply:
OK